sechub-wrapper-owasp-zap/src/test/java/com/mercedesbenz/sechub/owaspzapwrapper/config/OwaspZapScanContextFactoryTest.java

```diff
@@ -491,6 +491,22 @@ void includes_and_excludes_from_sechub_json_are_inside_result() {
         assertEquals(2, result.getOwaspZapURLsExcludeList().size());
     }
 
+    @ParameterizedTest
+    @CsvSource({ "true", "false" })
+    void connection_check_from_settings_is_in_result(boolean enabled) {
+        /* prepare */
+        CommandLineSettings settings = createSettingsMockWithNecessaryParts();
+        when(settings.isConnectionCheckEnabled()).thenReturn(enabled);
+        when(ruleProvider.fetchDeactivatedRuleReferences(any())).thenReturn(new DeactivatedRuleReferences());
+
+        /* execute */
+        OwaspZapScanContext result = factoryToTest.create(settings);
+
+        /* test */
+        assertEquals(result.isConnectionCheckEnabled(), enabled);
```

Comment:
It is enough to have `connectionCheckEnabled()` the `is…` as prefix is not necessary.

```suggestion
        assertEquals(result.connectionCheckEnabled(), enabled);
```